action.php

```diff
@@ -0,0 +1,54 @@
+<?php
+// CONNECT TO SERVER ---------------------------------------
+// require_once("connection.php");
+
+if (isset($_POST["submit"])) {
+    // ----------------------------------------  NAME AND CONTACTS  ----------------------------------------
```

Comment:
This isn't the correct format of adding comments. Plese adhere to the documentation provided and update accordingly